.vscode/settings.json

```diff
@@ -15,6 +15,6 @@
   },
   "cSpell.words": ["gemwallet"],
   "[typescriptreact]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
+    "editor.defaultFormatter": "vscode.typescript-language-features"
```

Comment:
What is the reason behind changing to this formater?

Reply:
Reverted